pystruct/learners/structured_perceptron.py

```diff
@@ -8,6 +8,10 @@ def inference(model, x, w):
     return model.inference(x, w)
 
 
+def inference_map(args):
```

Comment:
did you see my proposal of implementing a mapstar? That seems somewhat cleaner to me.